### PR TITLE
report: use fixed position for hidden radios

### PR DESCRIPTION
**Pull request by adamraine**
Fixes https://bugs.chromium.org/p/chromium/issues/detail?id=1439785

The absolute positioning of these radios caused the report rendering to bug out. Would be nice if there was a less hacky way to hide visually but still allow screen readers to use.

### CODE REVIEW

File: report/assets/styles.css [hunk 1]
```diff
@@ -670,7 +670,8 @@
 }
 
 .lh-metricfilter__radio {
-  position: absolute;
+  /* Instead of hiding, position of screen so it's still accessible to screen readers */
```
Comment (connorjclark) [inline report/assets/styles.css:673]:
```suggestion
  /* Instead of hiding, position offscreen so it's still accessible to screen readers */
```

**Comment by connorjclark** [comment]
Do we know why this happens?

Should we try to cherrypick the fix to 116 / 115?

**Review by connorjclark — APPROVED** [review]

File: report/assets/styles.css (hunk 1 above)
Comment (connorjclark) [reply report/assets/styles.css:673]: lets add a link to the crbug

**Comment by adamraine** [comment]
> Do we know why this happens?

Talked offline about this. Still not 100% sure on the root cause and may need to get help from DT rendering folks since this only happens within DT.

**Comment by paulirish** [comment]
> Would be nice if there was a less hacky way to hide visually but still allow screen readers to use.

The typical approach is this fairly nasty looking `clip` technique.  Nasty looking but.. it's used everrrrrywhere:

https://github.com/twbs/bootstrap/blob/v5.0.2/scss/mixins/_visually-hidden.scss#L3-L18
https://www.a11yproject.com/posts/how-to-hide-content/
https://github.com/csstools/sanitize.css/blob/092d0d85922bfa72d28e9e8d25d80a5437c8df44/sanitize.css#L344-L356
https://css-tricks.com/inclusively-hidden/

**Comment by adamraine** [comment]
> The typical approach is this fairly nasty looking clip technique. Nasty looking but.. it's used everrrrrywhere:

I took a look at similar solutions, however it was the `position: absolute` *not* the `left: -9999` which was causing issues.

**Pull request merged**